sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/debug/DebugConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.debug;
+
+/**
+ * Determines if the SDK is in debugging mode (captures stack traces) or not.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public final class DebugConfig {
+  private static final String ENABLE_METRICS_DEBUG_PROPERTY = "io.opentelemetry.sdk.metrics.debug";
+  private static boolean enabled;
+
+  private DebugConfig() {}
+
+  static {
+    // Should we also look for an environment variable?
```

Comment:
yes, at least for consistency with the rest of the configuration available

Reply:
It'd be nice if we could use `ConfigProperties` here. But the implementation (`DefaultConfigProperties`) is in the autoconfigure module and is package private.

Reply:
Attempted to unify with the logic.